impeller/aiks/canvas.cc

```diff
@@ -202,6 +203,10 @@ bool Canvas::AttemptDrawBlurredRRect(const Rect& rect,
       paint.mask_blur_descriptor->style != FilterContents::BlurStyle::kNormal) {
     return false;
   }
+  // A blur sigma that is close to zero should not result in any shadow.
+  if (std::fabs(paint.mask_blur_descriptor->sigma.sigma) <= kEhCloseEnough) {
+    return true;
+  }
```

Comment:
Could you add an early return to SolidRRectBlurContents too?

Because there's no guard against zero sigma, nothing is protecting us from div by zero in IPGaussian, so NaNs are probably propagating from the shader and getting mapped to a negative number on the float format we use for wide gamut.

Reply:
Good idea! Yes the nan issue, I remember it now. Thanks @bdero :)